metapackage_dependencies.rosinstall

```diff
@@ -0,0 +1,9 @@
+# This file should contain any source dependencies that are required for this metapackage to build. If the dependency is
```

Comment:
By `metapackage`, do you mean the entire workspace or catkin's definition of metapackage?

Reply:
By the way, I had added `moveit` and `moveit_msgs` to the rosinstall that used to exist in the root of the repo in my PR, I'm assuming that's gone now and it has been replaced with `upstream_dependencies.rosinstall`?

Reply:
yeah those two should be in the upstream dependencies. by `metapackage`, i'm referring to the `uwrt_mars_rover` metapkg that u clone into your ws. the contents of this repo is the metapkg.